code/modules/admin/verbs/one_click_antag.dm

```diff
@@ -522,6 +523,91 @@ client/proc/one_click_antag()
 
 	return
 
+/datum/admins/proc/makeERPsquad()
+	var/list/mob/dead/observer/candidates = list()
+	var/time_passed = world.time
+
+	//Generates a list of ERPers from active ghosts.
+	for(var/mob/dead/observer/G in player_list)
+		spawn(0)
+			switch(alert(G,"Do you wish to be considered for an elite Nanotrasen ERP Squad being sent in?","Please answer in 30 seconds!","Yes","No"))
+				if("Yes")
+					if((world.time-time_passed)>300)//If more than 30 game seconds passed.
+						return
+					candidates += G
+				if("No")
+					return
+				else
+					return
+	sleep(300)

```

Comment:
Jesus christ we need a better way to poll people about things than this sleep shit